docs/sources/datasources/mssql.md

```diff
@@ -16,7 +16,7 @@ Grafana ships with a built-in Microsoft SQL Server (MS SQL) data source plugin t
 | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
 | `Name`           | The data source name. This is how you refer to the data source in panels and queries.                                                 |
 | `Default`        | Default data source means that it will be pre-selected for new panels.                                                                |
-| `Host`           | The IP address/hostname and optional port of your MS SQL instance. If port is omitted, default 1433 will be used.                     |
+| `Host`           | The IP address/hostname and optional port of your MS SQL instance. If the port is omitted, the driver default will be used (0).       |
```

Comment:
```suggestion
| `Host`           | The IP address/hostname and optional port of your MS SQL instance. If the port is omitted, the driver default port 0 will be used.       |
```

Reply:
@achatterjee-grafana I'm a bit skeptical of that wording, because the driver won't necessary use port 0 in the end, but can translate to real one (0 means to make the choice automatically).

Reply:
Sounds good. I added a comment and not a change request. Thanks for the detailed explanation. It helps!